scripts/tests/test_zkai_attention_kv_d128_four_head_seq64_air_private_softmax_table_lookup_gate.py

```
@@ -0,0 +1,251 @@
+        return payload
+
+    def assert_rejects(self, payload, message):
+        with self.assertRaises(gate.AttentionKvAirPrivateSoftmaxTableLookupGateError) as ctx:
+            gate.validate_payload(payload, allow_missing_mutation_summary=True)
+        self.assertIn(message, str(ctx.exception))
+
+    def same_digit_mutation(self, value):
+        for candidate in (value + 1, value - 1):
+            if 0 <= candidate <= 255 and len(str(candidate)) == len(str(value)):
+                return candidate
+        self.fail(f"no same-digit mutation available for {value}")
```

Ruff (0.15.13)

[warning] Missing explicit `return` at the end of function able to return non-`None` value

Add explicit `return` statement

(RET503)